docs/API.md

```diff
@@ -409,6 +436,6 @@ The deviceCallbackPublicKey and deviceCallbackAuthKey fields are urlsafe-base64
                  d.name AS deviceName, d.type AS deviceType,
                  d.createdAt AS deviceCreatedAt, d.callbackURL AS deviceCallbackURL,
                  d.callbackPublicKey AS deviceCallbackPublicKey,
-                 d.callbackAuthKey AS deviceCallbackAuthKey
+                 d.callbackAuthKey AS deviceCallbackAuthKey, ut.tokenVerificationId
```

Comment:
Oh, I guess it's the "unverified tokens" table, but we should add it to the list of table abbreviations above (where we describe `t.*` and `a.*` etc.